common/definition/indexedKeys.go

```diff
@@ -52,14 +52,15 @@ const (
 	CadenceChangeVersion = "CadenceChangeVersion"
 )
 
-// valid non-indexed fields on ES
 const (
+	// Memo is valid non-indexed fields on ES
 	Memo = "Memo"
+	// Attr is prefix of custom search attributes
+	Attr = "Attr"
+	// HeaderFormat is the format of context headers in search attributes
+	HeaderFormat = "Header.%s"
```

Comment:
do we need to add validation somewhere to ensure that users don't create `Header.Something` search attributes?

Reply:
We use admin API `AddSearchAttribute` from admin CLI to onboard a new search attributes. If we block there, it's going to be manual to update schema on visibility storage directly, which is not ideal. 
The validation thus has to be manual